docs/node/migrations/zero-downtime-migrations/tutorial.mdx

```diff
@@ -0,0 +1,158 @@
+```
+```bash
+Migration Status:
+        Proposal ID: 4b0d1ad8-f9a5-5252-97e6-387d47b1ee9e
+        ExpiresAt: 100810
+        Approvals Received: 2 (needed 3)
+                Validator 8b90c1a6c017eb42f3c8496524bb186a1900b8b832309713e797d7f5defdccec: (approved)
+                Validator 594e07489b13bfcae681482423dc4327a2f99601f74b7be785501f1dd3c022be: (approved)
+                Validator 6d28361677bfe61fe86f3dd4457e069dfba57f912a5c01d554de528766f8a1c3: (not approved)
+                Validator b7404d70ae6f21b9f43d26fde6acc378be51c170a9744cb08634b6bdae22e1d2: (not approved)
+```
+
+If the proposal is approved by at least a supermajority of the validators, the network proposal is considered approved and the migration process is set to start after the activation period. 
+
+As the migration proposal is approved, the proposal is moved from pending to approved state, therefore the `kwil-admin migrate list/status` commands will not list the proposal anymore.
+
+```bash
+kwil-admin migrate status 4b0d1ad8-f9a5-5252-97e6-387d47b1ee9e --rpcserver /tmp/kwild_0.socket
+```
+```bash
+jsonrpc.Error: code = -400, message = failed to retrieve resolution, data = null
+err code = -400, msg = failed to retrieve resolution
+```
+However `kwil-admin node status` will show the migration status of the node as shown below:
+
+```bash
+kwil-admin node status --rpcserver /tmp/kwild_0.socket
+```
+```bash
+"migration": {
+    "status": "MigrationNotStarted",
+    "start_height": 143,
+    "end_height": 2143
+}
+```
+
+## Genesis State
+
+Once the proposal is approved, the network is set to start the migration process after the specified activation period. The snapshots and genesis files generated at the start of the migration can be downloaded using the `kwil-admin migrate genesis-state` command as shown below.
+
+```bash
+kwil-admin migrate genesis-state --root-dir /tmp/chain2  --rpcserver /tmp/kwild_1.socket
+```
+
+Note that the command will not download any files if there is no approved migration or the migration has not yet started. Please retry the command after the migration is activated.
+
+```bash
+No genesis state to download yet. Migration is set to start at block height: 143
+```
+
+:::warning
+The `genesis.json` and `snapshot.sql.gz` files are generated in the directory specified using the `--root-dir` flag. Ensure that you don't run the command with `--root-dir` flag set to the current node's root directory as it can overwrite the existing genesis file in the node's root directory.
+
+If the admin service is accessible from the new node it can be run on the new node directly.
+:::
+
+```bash
+kwil-admin migrate genesis-state --root-dir /tmp/chain2  --rpcserver /tmp/kwild_1.socket
+Migration State:
+        Start Height: 143
+        End Height: 2143
+        Downloaded Genesis File: .testnet2/genesis.json
+        Downloaded Snapshot File: .testnet2/snapshot.sql.gz
+```
+:::info
```

Comment:
Although Docusaurus doesn't care, markdown should have blank lines between all code fences and other mdx block, and below section headers.